src/ls_server.rs

```diff
@@ -488,7 +486,6 @@ pub trait Output {
 
     fn notify(&self, message: &str) {
         let output = serde_json::to_string(&NotificationMessage {
-            jsonrpc: "2.0".to_owned(),
```

Comment:
Here and above. I don't think you can just remove these "2.0" strings, we still need to write these out some how. While success does this 'manually' for notify and failure we do it via serialisation. It seems with this change you are not doing it at all.